src/Tippy.js

```diff
@@ -79,6 +84,18 @@ function Tippy(props) {
     }
   })
 
+  useEffect(() => {
```

Comment:
This looks good. I think there should be an extra line after the isMounted block (line 90-91)

Reply:
Also, I think this should actually be `useLayoutEffect`. Reason being, mutations to the DOM (tippy element in this case) need to be synchronous with every render, and not wait after paint. It might cause undesirable transitions or something?